MessengerOnFileForLinux/ChatControl/src/ConversationControl.hpp

```diff
@@ -19,8 +20,15 @@ class ConversationControl
 public:
     void conversation();
     void conversationEpilog();
+
     ConversationControl(std::shared_ptr<ChatInformation> chatInfo);
+    ConversationControl() = delete;
```

Comment:
to bym przesunał pod destruktor, odzielił delety enterem. Tak jak jest w Controler.hpp

Reply:
+ czy potrzebujemy  ConversationControl() = delete, skoro i tak nie wywołamy domyslengo konstruktora jeśli mamy zdefiniowany jakis z argumentem?

Reply:
Masz rację, inaczej próbuje zmatchować (&) lub (&&) z pustym argumentem, wię wystarczy usunięcie tamtych.